DocumentDataAPI/DocumentDataAPITests/Data/Repositories/NpgDocumentRepositoryIntegrationTests.cs

```diff
@@ -19,7 +18,45 @@ public NpgDocumentRepositoryIntegrationTests()
         _logger = new Logger<NpgDocumentRepository>(new NullLoggerFactory());
         TestHelper.DeployDatabaseWithTestData();
     }
+    
+    [Fact]
+    public void Get_ReturnsCorrectRows()
+    {
+        // Arrange
+        NpgDocumentRepository repository = new(_connectionFactory, _logger);
+        DocumentModel model = new("Maja Lærke Maach",
+                                  DateTime.Parse("2022-10-07T13:40:00"),
+                                  1,
+                                  "https://www.dr.dk/nyheder/seneste/iran-haevder-mahsa-amini-doede-af-organsvigt",
+                                  1,
+                                  "",
+                                  "Iran hævder, at Mahsa Amini døde af organsvigt",
+                                  0);
+        int id = 1;
+
+        // Act
+        DocumentModel? result = repository.Get(id);
+        
+        // Assert
+        result.Should().NotBeNull();
+        result.Should().BeEquivalentTo(model);
```

Comment:
Kan (bør?) chaines sammen: `result.Should().x1.And.x2`. Dunno om det gør en forskel tho, maybe not

Reply:
Tænkte jeg også over, men det er lidt mere readable. Idk

Reply:
Jeg kan huske, at vi fik at vide i OOP, at man kun skulle assert én gang per test.

Reply:
True. Men altså at have should().x1.and.x2 er basically to asserts

Reply:
Ja og nej. Det ville være den rigtige måde at gøre det på. For det har noget at gøre med om den anden test bliver kørt baseret på den første.